### PR TITLE
New package: mpd-sima

### PR DESCRIPTION
**Pull request by felix**

### CODE REVIEW

File: srcpkgs/mpd-sima/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+depends="python3-musicpd>=0.4.0 python3-requests>=2.2.0"
+short_desc="Automagically add titles to MPD playlist"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="GPL-3"
+homepage="http://kaliko.me/code/mpd-sima"
+distfiles="http://media.kaliko.me/src/sima/releases/MPD_sima-${version}.tar.xz"
+checksum=a19d5cecd5bac13628648417e655fce052702b44a248a33192395a336415a506
+
+post_install() {
+	vlicense doc/COPYING.GPL
```
Comment (Vaelatern) [inline srcpkgs/mpd-sima/template]: No need because GPL

File: srcpkgs/python3-musicpd/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+hostmakedepends="python3-setuptools "
+short_desc="An MPD (Music Player Daemon) client library for Python3"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="GPL-3"
+homepage="http://kaliko.me/code/python-musicpd"
+distfiles="${PYPI_SITE}/p/python-musicpd/python-musicpd-${version}.tar.gz"
+checksum=ce9b648921c16bbd05c622cc6c915d3e86e4c97373d2b2be8222c5679890045b
+
+post_install() {
+	vlicense LICENSE.txt
```
Comment (Vaelatern) [inline srcpkgs/python3-musicpd/template]: GPL, again, no need.

File: srcpkgs/python3-musicpd/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=python3-musicpd
+version=0.4.2
+revision=1
+noarch=yes
+wrksrc="python-musicpd-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools "
+short_desc="An MPD (Music Player Daemon) client library for Python3"
+maintainer="Felix Hanley <felix@userspace.com.au>"
+license="GPL-3"
```
Comment (Vaelatern) [inline srcpkgs/python3-musicpd/template]: Not GPL-3. LGPL-3

File: srcpkgs/python3-musicpd/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'python-musicpd'
```
Comment (Vaelatern) [inline srcpkgs/python3-musicpd/template]: Please match packagename.

File: srcpkgs/python3-musicpd/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=python3-musicpd
+version=0.4.2
+revision=1
+noarch=yes
+wrksrc="python-musicpd-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools "
```
Comment (Vaelatern) [inline srcpkgs/python3-musicpd/template]: Extra space, just remove that if you please :)

**Comment by Vaelatern** [comment]
Okay, this single PR needs two commits in it. The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git).

**Comment by felix** [comment]
@Vaelatern update PR to have two commits

**Pull request merged**